src/components/ContentPreview/ContentPreview.js

```diff
@@ -256,7 +256,7 @@ class ContentPreview extends PureComponent<Props, State> {
 
     static getDerivedStateFromProps(props: Props, state: State) {
         const { fileId } = props;
-        if (fileId !== state.prevFileId && fileId !== state.currentFileId) {
+        if (fileId !== state.prevFileId) {
```

Comment:
Seems like this component might be simplified by using a `key` based on the fileId prop?

Reply:
It could definitely be simplified by doing this, but using ```key``` would cause the component to unmount and remount

Reply:
Ah, okay. In that case, could we switch from using `PureComponent` to checking if the fileId changed in `shouldComponentUpdate`? This can happen in a follow-up PR, as well; fixing the bug is the higher priority.